content/docs/csidriver/installation/test/certcsi.md

```diff
@@ -62,313 +147,524 @@ storageClasses:
+9. Mounts the raw block volume and verifes the checksum of the data.
 
-To run scalability test suite, run the command:
 ```bash
-cert-csi test scaling --sc <storage class> --replicas 5
+cert-csi test blocksnap --sc <storageClass> --vsc <snapshotclass>
 ```
 
-#### Running VolumeIO test suite
+Run `cert-csi test blocksnap -h` for more options.
+
+#### Volume Health Metrics
+1. Creates the namespace `volume-health-metrics-*` where resources will be created.
+2. Creates Persistent Volume Claim.
+3. Creates Pod to consume the Persistent Volume Claim.
+4. Waits for Pod to be in the Ready state.
+4. Veries that ControllerGetVolume and NodeGetVolumeStats are being executed in the controller and node pods, respectively.
 
-To run volumeIO test suite, run the command:
 ```bash
-cert-csi test vio --sc <storage class> --chainNumber 5 --chainLength 20
+cert-csi test volumehealthmetrics --sc <storage-class> --driver-ns <driver-namespace>
 ```
 
-#### Running Snap test suite
+Run `cert-csi test volumehealthmetrics -h` for more options.
 
-To run volume snapshot test suite, run the command:
-```bash
-cert-csi test snap --sc <storage class> --vsc <volume snapshot class> 
-```
+> Note: Make sure to enable healthMonitor for the driver's controller and node pods before running this suite. It is recommended to use a smaller interval time for this sidecar.
 
-#### Running Multi-attach volume suite
+#### Ephemeral Volumes
+1. Creates namespace `functional-test` where resources will be created.
+2. Creates Pods with one ephemeral inline volume each.
+3. Waits for Pods to be in the Ready state.
+4. Writes data to the volume on each Pod.
+5. Verifies the checksum of the data.
 
-To run multi-attach volume test suite, run the command: 
 ```bash
-cert-csi test multi-attach-vol --sc <storage class> --podNum 3
+cert-csi test ephemeral-volume --driver <driver-name> --attr ephemeral-config.properties
 ```
-```bash
 
-cert-csi test multi-attach-vol --sc <storage class> --podNum 3 --block # to use raw block volumes
-```
+Run `cert-csi test ephemeral-volume -h` for more options.
 
-#### Running Replication test suite
+> `--driver` is the name of a CSI Driver from the output of `kubectl get csidriver` (e.g, csi-vxflexos.dellemc.com).
+> This suite does not delete resources on success.
 
-To run replication test suite, run the command:
-```bash
+Sample ephemeral-config.properties (key/value pair)
+   {{< tabs name="volume-attributes-examples" >}}
+   {{% tab name="CSI PowerFlex" %}}
+
+   ```yaml
+   volumeName: "my-ephemeral-vol"
+   size: "10Gi"
+   storagepool: "sample"
+   systemID: "sample"
+   ```
 
-cert-csi test replication --sc <storage class> --pn 1 --vn 5 --vsc <snapshot class> 
-```
+   {{% /tab %}}
+   {{% tab name="CSI PowerScale" %}}
 
-#### Running Volume Cloning test suite
+   ```yaml
+   size: "10Gi"
+   ClusterName: "sample"
+   AccessZone: "sample"
+   IsiPath: "/ifs/data/sample"
+   IsiVolumePathPermissions: "0777"
+   AzServiceIP: "192.168.2.1"
```

Comment:
IP?

Reply:
It's a placeholder copied from the storage class example.

https://github.com/dell/csi-powerscale/blob/3a90ab76d2f1d77190565c0caab4330e75f2dd5b/samples/storageclass/isilon.yaml#L49